java/runtime/src/main/java/io/ray/runtime/gcs/GlobalStateAccessor.java

```diff
@@ -83,6 +83,13 @@ private void validateGlobalStateAccessorPointer() {
     }
   }
 
+  public byte[] getPlacementGroupInfo(String name, boolean global) {
```

Comment:
Overloded methods should all have same return type. Would it be better to name it as `getPlacementGroup`?

Reply:
sorry, I didn't get what's your mean, It seems that these methods are all return `byte[]`, right?  and the naming of `getxxxInfo` also just follow the previous design like `getNodeResourceInfo`、 `getAllNodeInfo` and `getAllJobInfo`

Reply:
My fault, I took `GlobalStateAccessor#getPlacementGroupInfo` as `GcsClient#getPlacementGroupInfo` by mistake